### ISSUE TITLE
Custom fragmet

### ISSUE THREAD

**Issue by rezaiyan** [Krupen/FabulousFilter#5]
how can add to viewpager custom fragmets ?
i've added some framgent but give me some error:
`
java.lang.IllegalStateException: Fragment does not have a view
...
`

**Comment by rezaiyan** [comment]
i think we can't add fragments to a view that parent is so fragment!!!

**Comment by Krupen** [comment]
Hi,
Nested fragments are only supported from API 17 (Jellybean).
I am not sure if that is the problem, if possible, please share your code or create a sample repository reproducing this behaviour so that I can solve it.

**Comment by rezaiyan** [comment]
By default i have a ViewPager and i've create some fragments with specific .xml layouts 
and i've added those fragments to viewpagerAdapter and setAdapter viewpager
in this case we have this error:
`
java.lang.IllegalStateException: Fragment does not have a view ...
`

**Comment by Krupen** [comment]
Yes, I tried it and it din't work, got the same error. Mostly this is because of nesting fragments. Will do more research on this and find a solution or workaround.

**Comment by rezaiyan** [comment]
Thank you so much! 


_Sent from my Xiaomi MI 3W using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by ZhenisMadiyar** [comment]
Who solve this problem? )

**Comment by Krupen** [comment]
@ZhenisMadiyar 
This problem is not yet solved. The workaround is to use ViewPager with Views instead of Fragments.

**Comment by Krupen** [comment]
Here is the reason well explained as to why FragmentPagerAdapter is not working inside BottomSheet:
https://stackoverflow.com/a/21418952

**Comment by divyank00** [comment]
Is it solved?
